browser/ui/views/sidebar/sidebar_container_view.cc

```diff
@@ -125,13 +125,18 @@ void SidebarContainerView::Init() {
   // Hide by default. Visibility will be controlled by show options later.
   DoHideSidebar(false);
   UpdateToolbarButtonVisibility();
+  SetSidebarOnLeft(sidebar_on_left_);
 }
 
 void SidebarContainerView::SetSidebarOnLeft(bool sidebar_on_left) {
   sidebar_on_left_ = sidebar_on_left;
```

Comment:
nit: Can we have `if (sidebar_on_left_ == sidebar_on_left) return; ` to avoid unncessasary layoutting?

Reply:
Added.